src/setup-open.ts

```diff
@@ -96,6 +94,12 @@ export function setupOpen(QuickSQLite: ISQLite) {
 
       const listenerManager = new DBListenerManagerInternal({ dbName });
 
+      // A incrementing integer ID for tracking lock requests
+      let requestIdCounter = 1;
```

Comment:
`LockCallbacks` is global, so this should also be global, to avoid multiple db connections having overlapping ids.

Reply:
Good catch! Updated it now.